docs/layouts/shortcodes/generated/forst_configurable_configuration.html

```diff
@@ -0,0 +1,168 @@
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.rescaling.use-delete-files-in-range</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>If true, during rescaling, the deleteFilesInRange API will be invoked to clean up the useless files so that local disk space can be reclaimed more promptly.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.restore-overlap-fraction-threshold</h5></td>
+            <td style="word-wrap: break-word;">0.0</td>
+            <td>Double</td>
+            <td>The threshold of overlap fraction between the handle's key-group range and target key-group range. When restore base DB, only the handle which overlap fraction greater than or equal to threshold has a chance to be an initial handle. The default value is 0.0, there is always a handle will be selected for initialization. </td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.thread.num</h5></td>
+            <td style="word-wrap: break-word;">2</td>
+            <td>Integer</td>
+            <td>The maximum number of concurrent background flush and compaction jobs (per stateful operator). The default value is '2'.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.use-bloom-filter</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>If true, every newly created SST file will contain a Bloom filter. It is disabled by default.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.use-ingest-db-restore-mode</h5></td>
+            <td style="word-wrap: break-word;">false</td>
+            <td>Boolean</td>
+            <td>A recovery mode that directly clips and ingests multiple DBs during state recovery if the keys in the SST files does not exceed the declared key-group range.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.write-batch-size</h5></td>
+            <td style="word-wrap: break-word;">2 mb</td>
+            <td>MemorySize</td>
+            <td>The max size of the consumed memory for ForSt batch write, will flush just based on item count if this config set to 0.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.writebuffer.count</h5></td>
+            <td style="word-wrap: break-word;">2</td>
+            <td>Integer</td>
+            <td>The maximum number of write buffers that are built up in memory. The default value is '2'.</td>
+        </tr>
+        <tr>
+            <td><h5>state.backend.forst.writebuffer.number-to-merge</h5></td>
+            <td style="word-wrap: break-word;">1</td>
+            <td>Integer</td>
+            <td>The minimum number of write buffers that will be merged together before writing to storage. The default value is '1'.</td>
```

Comment:
guidance on when to set

Reply:
This will be introduced in the ForSt project.